cabot/rest_urls.py

```diff
@@ -8,20 +8,30 @@
 router = routers.DefaultRouter()
 
 
```

Comment:
I made a bunch of cleanup changes to this file.  Not entirely necessary.  They are:

- Add comments to `create_viewset()`
- Use dashes instead of underscores in API names (eg: `status-checks` instead of `status_checks`).  More web-friendly.  Does anything currently use these endpoints?